Sources/ProjectAutomation/Tuist.swift

```diff
@@ -25,19 +25,8 @@ public enum Tuist {
         }
     }
 
-    /// Returns graph at the current path.
-    public static func graph() throws -> Graph {
-        try graph(at: nil)
-    }
-
-    /// Returns graph at the given path.
-    public static func graph(at path: String) throws -> Graph {
-        try graph(at: path as String?)
-    }
-
-    // MARK: - Helpers
-
-    private static func graph(at path: String?) throws -> Graph {
+    /// Returns the graph at the given path.
```

Comment:
Could you add an info about expected behavior when the path is equal to `nil`?

"The current path will be used" or something similar to.